docs/contributing/code-contributions.md

```diff
@@ -119,6 +120,21 @@ Using Docker Compose, you can start and stop a WordPress instance and integrate
 
 Check [Debugging the build process](/docs/debugging-the-build-process/) page to learn how to debug Gatsby.
 
+## Official theme development
+
+This section is for official theme development in Gatsby's monorepo. If you are looking
+to build your own theme, see [building themes](/docs/themes/building-themes/).
+
+Before getting started, make sure that you have
+[set up your local dev environment](/contributing/setting-up-your-local-dev-environment/)
+and that you're on the latest version of `gatsby-dev-cli`.
+
+- In the Gatsby monorepo, change directory to one of the starters using the theme you want to work on: `cd starters/gatsby-starter-blog-theme`
+- Install dependencies: `yarn`
+- Run Gatsby Dev CLI to sync theme files: `gatsby-dev --packages gatsby-theme-blog`
+- In another tab run the starter: `yarn develop`
```

Comment:
This is failing for me. On sharp though, so hard to say if it's just my local stuff.

Reply:
Got it! Looks like it was a weird node thing.